### PR TITLE
Add Calendar page

### PR DESCRIPTION
**Pull request by ct-martin**
Based on RITlug site calendar. I leave it to you to choose where to link (outside this PR).

### CODE REVIEW

**Comment by jwflory** [comment]
Thanks @ct-martin. I'll try to review this by the end of the week. I also requested review by the Tech Team since I am not particularly web-savvy.

**Comment by Tjzabel** [comment]
This is essentially verbatim the RITlug website PR that we merged a few days ago. You can see the merged view [here](https://ritlug.com/calendar/).

I'll leave it to you guys to approve.

**Comment by ct-martin** [comment]
@jwflory 

1. I asked you offline at BrickHack, but calendar is based on well-formed metadata (frontmatter) of posts. Since this isn't specific to the calendar and is more of documentation on how to write a post, would you like it in `README.md` or `CONTRIBUTING.md`? Also, I am of the opinion that it's better to keep this information in the repo than the runbook due to the fact there there are implementation details in the explanations. See [the RITlug README](https://github.com/RITlug/ritlug.github.io/blob/master/README.md#adding-content--contributing) for what this would look like
2. We've actually hit the limit of how much as can fit in the navbar, do you mind merging this and continuing the discussion in another PR? I doubt you want to remove anything, but adding dropdowns to the navbar will be beyond the scope of this PR.
Screen Shot 2020-02-13 at 12 09 28

**Comment by jwflory** [comment]
Hey @ct-martin, could you please rebase this PR on the latest `master` branch? I'll document the how-to piece in the Runbook along with the other website content and we can move this PR along. Thanks for being patient.

**Comment by ct-martin** [comment]
@jwflory done. In particular, please note the `date-start` and `date-end` frontmatter when you're updating the runbook (this is in the RITlug docs)

File: meetings-meetups/_posts/2020-01-16-foss-hours.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: redirect
+redirect: https://fossrit.github.io/
+date-start: "2020-01-16 17:00"
+date-end: "2020-01-16 19:00"
+location: "MSS-3190"
+title: "FOSS Hours"
+rrule: "FREQ=WEEKLY;UNTIL=20200423T190000"
```
Comment (jwflory) [inline meetings-meetups/_posts/2020-01-16-foss-hours.md]: What is the syntax for this frontmatter? I want to link out to documentation about how to write these for future contributors.

Comment (ct-martin) [reply meetings-meetups/_posts/2020-01-16-foss-hours.md]: @jwflory https://github.com/RITlug/ritlug.github.io#calendar--placeholders

> Placeholders can also take an `rrule` frontmatter for recurring events, written [to the iCal spec](https://icalendar.org/iCalendar-RFC-5545/3-3-10-recurrence-rule.html).
>
> Of particular note,
> 
> > The UNTIL rule part defines a DATE or DATE-TIME value that bounds the recurrence rule in an inclusive manner. If the value specified by UNTIL is synchronized with the specified recurrence, this DATE or DATE-TIME becomes the last instance of the recurrence. The value of the UNTIL rule part MUST have the same value type as the "DTSTART" property.

Comment (jwflory) [reply meetings-meetups/_posts/2020-01-16-foss-hours.md]: @ct-martin Thanks. I'll use that as a base for the Runbook.

**Pull request merged**